lib/notification_report.rb

```diff
@@ -1,3 +1,5 @@
+require 'csv'
```

Comment:
This file is already using double quotes. We should be consistent (also on lines 135, 154)